tasks/get-variables/get-variables.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: get-variables
```

Comment:
@ShakedAviv50313 please change the name to `parse-build-deploy-script` as mentioned in the jira: https://issues.redhat.com/browse/RHTAP-2076